### ISSUE TITLE
Sort: Use consistent terminology for sorting the notes list

### ISSUE THREAD

**Issue by rachelmcr** [Automattic/Simplenote-United#42]
### What

It would help to have all the Simplenote apps use the same terminology for sorting the notes list. For example, iOS uses "Newest"/"Oldest" for dates (modified or created) and "A-Z"/"Z-A" for alphabetical ordering. The web uses "Ascending"/"Descending" for all of the Sort By options.

I'd suggest using "Newest"/"Oldest" and "A-Z"/"Z-A" terminology across the apps.

### Why

It takes more mental effort to determine what "Ascending" and "Descending" mean for sorting, and in any case it would be nice to have consistent terminology across all the apps.

Tested in Simplenote iOS 4.8.0 beta and app.simplenote.com.

**Comment by SylvesterWilmott** [comment]
>I'd suggest using "Newest"/"Oldest" and "A-Z"/"Z-A" terminology across the apps.

Agreed. This also reflects what was done for note organisation project. Thanks @rachelmcr

**Comment by codebykat** [comment]
This isn't super relevant anymore since web now uses (the probably equally oblique) "Reversed":

<img width="447" alt="Screen Shot 2020-09-29 at 12 49 42 AM" src="https://user-images.githubusercontent.com/52152/94528760-b1fda880-01ed-11eb-8b33-4cad712c8926.png">

Here's [an idea I had about it](https://github.com/Automattic/simplenote-electron/tree/update/clarify-sort-in-settings), but I'm not sure about it since I don't think "Z-A" would localize well. How does iOS handle this in other languages?

<img width="443" alt="Screen Shot 2020-09-29 at 12 47 58 AM" src="https://user-images.githubusercontent.com/52152/94528879-dbb6cf80-01ed-11eb-96c3-11d4b74c08a7.png">

<img width="414" alt="Screen Shot 2020-09-29 at 12 48 06 AM" src="https://user-images.githubusercontent.com/52152/94528895-e07b8380-01ed-11eb-9bf1-ae7aeee572f7.png">

For reference here's what iOS currently looks like:
![IMG_6778 (2)](https://user-images.githubusercontent.com/52152/94530888-9b0c8580-01f0-11eb-845b-34ea799c554a.jpeg)
